clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookResolveURITest.spec.ts

```diff
@@ -93,7 +93,7 @@ async function buildResultArrayFrom(
   );
   return result.length;
 }
-
+Utils.getExtensionFolder = jest.fn().mockReturnValue(__dirname);
```

Comment:
same here.

Reply:
this is mocked for : `searchCopybookInExtensionFolder` there is an indirect call for this.

Reply:
removed at: 533bc9f181e4bd6b8c63959229b92a9797bcbb99